SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-intro.md

```diff
@@ -4,9 +4,9 @@
 <%(Callout type="warn" message="This tutorial is available as a preview to demonstrate early multiserver functionality in the GDK. Please note we do not yet recommend development of multiserver games. For more information, please follow our [development roadmap](https://github.com/spatialos/UnrealGDK/projects/1) and [Unreal features support]({{urlRoot}}/unreal-features-support) pages.")%>	
 
 **What the tutorial covers**<br/>
-In this tutorial you’ll implement cross-server remote procedure calls (RPCs) in a simple third person shooter. The end result will be a multiplayer, cloud-hosted Unreal game running across multiple [server-workers]({{urlRoot}}/content/glossary#inspector) that players can seamlessly move between and shoot across. It will look something like this:
+In this tutorial you’ll implement cross-server remote procedure calls (RPCs) in a simple first person shooter example project. The end result will be a multiplayer, cloud-hosted Unreal game running across multiple [server-workers]({{urlRoot}}/content/glossary#inspector) that players can seamlessly move between and shoot across. It will look something like this:
```

Comment:
```suggestion
>**This tutorial uses the Example Project from the GDK’s [setup guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro)**.

In this tutorial you’ll implement cross-server remote procedure calls (RPCs) for simple first-person shooter functionality in the Example Project. The end result will be a multiplayer, cloud-hosted Unreal game running across multiple [server-workers]({{urlRoot}}/content/glossary#worker) that players can seamlessly move between and shoot across. It will look something like this:
```

This is how we introduce the multi deployment tutorial - https://docs.improbable.io/unreal/alpha/content/tutorials/deployment-manager/tutorial-deploymentmgr-intro